frontend/.env.example

```
@@ -102,4 +102,6 @@ TRUST_FORWARDED_HEADERS=0
 # emergency switch
 RATE_LIMIT_DISABLED=0
 
-GROQ_API_KEY=
\ No newline at end of file
+GROQ_API_KEY=
+
+NEXT_PUBLIC_WEB3FORMS_KEY=
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)